types/vscode-proposed/index.d.ts

```diff
@@ -267,12 +267,6 @@ export interface NotebookEditor {
      */
     readonly document: NotebookDocument;
 
-    /**
```

Comment:
I pulled in just the selection change here. Pulling in the file was bringing in more kernel deprecations that I'd rather deal with the kernel push work.

@DonJayamanne Side note, we seem to have always had three copies of this file. Can we condense down to one? Wasn't sure if there was a reason for the copies, so I didn't do that in this PR.

Reply:
I don't recall why we had 3, possibly something from core extension. Or maybe when we were using ts-node or the like...
Please do feel free to remove the duplicates.

Reply:
@DonJayamanne sounds good. Will do with kernel API update.